.github/ISSUE_TEMPLATE/feature.yml

```
@@ -0,0 +1,32 @@
+name: Feature Request
+description: File a feature request
+title: ":pray: "
```

```suggestion
title: ":pray: "
```
What do you think about using 🗳️ here? It gets around the default yellow emoji hands which is a personal pet peeve of mine

---

Absolutely, I'll update this!